projects/plugins/jetpack/extensions/blocks/videopress/v6/edit.js

```diff
@@ -225,7 +248,18 @@ export default function VideoPressEdit( { attributes, setAttributes } ) {
 					// eslint-disable-next-line no-console
 					console.error( 'Error: ', error );
 				} }
-			/>
+			>
+				{ uploadErrorData && (
+					<div
+						role="alert"
+						aria-live="assertive"
+						className="jetpack-videopress-upload-error-message"
+					>
```

Comment:
I think we use to use [a Notice instance](https://wordpress.github.io/gutenberg/?path=/story/components-notice--default&knob-Icon=smiley&knob-Instructions=Here%20are%20instructions%20you%20should%20follow&knob-Label=My%20Placeholder%20Label&knob-isColumnLayout=true) to render these kind of messages

Reply:
@retrofox At first I used notices, but It stayed on top of the block, didn't like it this way. I prefer it on the bottom.

Reply:
Anyway and IMO, what we should do is follow the designs done in core or other implementations, at least until we have specific designs to implement. It seems to be a pretty random design which is ok since we are in the middle of the development process, meaning that we'll probably iterate over it soon.